### ISSUE TITLE
plugin.tx_powermail.settings.setup.receiver.overwrite.senderEmail is ignored

### ISSUE THREAD

**Issue by matcolton** [einpraegsam/powermail#777]
TYPO3 11.5.10
powermail 10.3.2
PHP 7.4

The following setup options to not work anymore:
<pre><code>plugin.tx_powermail.settings.setup.receiver.overwrite.senderEmail = TEXT
plugin.tx_powermail.settings.setup.receiver.overwrite.senderEmail.value = myemail@mydomain.tld
</code></pre>

I have seen the issue list [#761](https://github.com/einpraegsam/powermail/issues/761), but I'm not quite sure if this is related. 

This is not an issue of SPF itself. As far as I can tell the overwrite values are not even checked for the reciever `senderEmail`. In the method `getSenderEmail()` in `ReceiverMailSenderPropertiesService.php` it is only checked against `getSenderMailFromArguments()`. And in `getSenderMailFromArguments()` it is only checked if any field has been set as sender email. If not the default values are used. The overwrite values are not taken into account.

**Comment by einpraegsam** [comment]
I just checked in the latest powermail version and overwriting works as expected:

```
plugin.tx_powermail.settings.setup.receiver.overwrite.senderEmail = TEXT
plugin.tx_powermail.settings.setup.receiver.overwrite.senderEmail.value = lalelu@einpraegsam.net
```
![overwrite](https://user-images.githubusercontent.com/89055/172835832-36ccef57-302b-4879-a45c-f77661cfc579.png)

Any other hints for me?

**Comment by matcolton** [comment]
Thanks for the lightning fast response. 

I deactivated all other extensions (container, dce, content_defender, mindshape_cookie_consent, staticfilecache) except powermail and the siteconfig. It then worked flawlessly. So I enabled one extension after the other. Now all extensions are active again and it works as expected with no changes at all. 

I guess it was a cache issue. But I really don't understand why, as I cleared the cache via the installtool multiple times including dumping the autoload before filing this report. 

Sorry for bugging you and keep up the great work!
